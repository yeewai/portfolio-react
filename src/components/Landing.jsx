import React from 'react';
import { Link } from 'react-router-dom';

const Landing = () => (
    <section className="landing" style={{"margin-top": `${(window.innerHeight - 600)/2.2}px`}}>
        <nav>
            <div className="row justify-content-around">
                <div className="col" id="art" >
                    <Link to="/art" className="nav animated fadeInUp">Art</Link>
                </div>
                <div className="col-2" id="about" >
                    <Link to="/about" className="nav animated fadeIn animation-delay-1000">Yee Wai</Link>
                </div>
                <div className="col" id="code" >
                    <Link to="/code"className="nav animated fadeInUp">Code</Link>
                </div>
            </div>
        </nav>
    </section>
)

export default Landing;
