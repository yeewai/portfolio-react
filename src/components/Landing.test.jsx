import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Landing from './Landing';

Enzyme.configure({ adapter: new Adapter() });

describe("Landing Page", () => {
    it('renders', () => {
      const wrapper = shallow(<Landing />);
      expect(wrapper).toMatchSnapshot();
    });
});
