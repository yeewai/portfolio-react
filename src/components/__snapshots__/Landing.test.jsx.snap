// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav>
        <div
                className="row justify-content-around"
        >
                <div
                        className="col"
                        id="art"
                >
                        <Link
                                className="nav animated fadeInUp"
                                replace={false}
                                to="/art"
                        >
                                Art
                        </Link>
                </div>
                <div
                        className="col-2"
                        id="about"
                >
                        <Link
                                className="nav animated fadeIn animation-delay-1000"
                                replace={false}
                                to="/about"
                        >
                                Yee Wai
                        </Link>
                </div>
                <div
                        className="col"
                        id="code"
                >
                        <Link
                                className="nav animated fadeInUp"
                                replace={false}
                                to="/code"
                        >
                                Code
                        </Link>
                </div>
        </div>
</nav>,
      "className": "landing",
      "style": Object {
        "margin-top": "76.36363636363636px",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row justify-content-around"
>
          <div
                    className="col"
                    id="art"
          >
                    <Link
                              className="nav animated fadeInUp"
                              replace={false}
                              to="/art"
                    >
                              Art
                    </Link>
          </div>
          <div
                    className="col-2"
                    id="about"
          >
                    <Link
                              className="nav animated fadeIn animation-delay-1000"
                              replace={false}
                              to="/about"
                    >
                              Yee Wai
                    </Link>
          </div>
          <div
                    className="col"
                    id="code"
          >
                    <Link
                              className="nav animated fadeInUp"
                              replace={false}
                              to="/code"
                    >
                              Code
                    </Link>
          </div>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col"
              id="art"
>
              <Link
                            className="nav animated fadeInUp"
                            replace={false}
                            to="/art"
              >
                            Art
              </Link>
</div>,
            <div
              className="col-2"
              id="about"
>
              <Link
                            className="nav animated fadeIn animation-delay-1000"
                            replace={false}
                            to="/about"
              >
                            Yee Wai
              </Link>
</div>,
            <div
              className="col"
              id="code"
>
              <Link
                            className="nav animated fadeInUp"
                            replace={false}
                            to="/code"
              >
                            Code
              </Link>
</div>,
          ],
          "className": "row justify-content-around",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="nav animated fadeInUp"
                replace={false}
                to="/art"
>
                Art
</Link>,
              "className": "col",
              "id": "art",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Art",
                "className": "nav animated fadeInUp",
                "replace": false,
                "to": "/art",
              },
              "ref": null,
              "rendered": "Art",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="nav animated fadeIn animation-delay-1000"
                replace={false}
                to="/about"
>
                Yee Wai
</Link>,
              "className": "col-2",
              "id": "about",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Yee Wai",
                "className": "nav animated fadeIn animation-delay-1000",
                "replace": false,
                "to": "/about",
              },
              "ref": null,
              "rendered": "Yee Wai",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="nav animated fadeInUp"
                replace={false}
                to="/code"
>
                Code
</Link>,
              "className": "col",
              "id": "code",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Code",
                "className": "nav animated fadeInUp",
                "replace": false,
                "to": "/code",
              },
              "ref": null,
              "rendered": "Code",
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav>
          <div
                    className="row justify-content-around"
          >
                    <div
                              className="col"
                              id="art"
                    >
                              <Link
                                        className="nav animated fadeInUp"
                                        replace={false}
                                        to="/art"
                              >
                                        Art
                              </Link>
                    </div>
                    <div
                              className="col-2"
                              id="about"
                    >
                              <Link
                                        className="nav animated fadeIn animation-delay-1000"
                                        replace={false}
                                        to="/about"
                              >
                                        Yee Wai
                              </Link>
                    </div>
                    <div
                              className="col"
                              id="code"
                    >
                              <Link
                                        className="nav animated fadeInUp"
                                        replace={false}
                                        to="/code"
                              >
                                        Code
                              </Link>
                    </div>
          </div>
</nav>,
        "className": "landing",
        "style": Object {
          "margin-top": "76.36363636363636px",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row justify-content-around"
>
            <div
                        className="col"
                        id="art"
            >
                        <Link
                                    className="nav animated fadeInUp"
                                    replace={false}
                                    to="/art"
                        >
                                    Art
                        </Link>
            </div>
            <div
                        className="col-2"
                        id="about"
            >
                        <Link
                                    className="nav animated fadeIn animation-delay-1000"
                                    replace={false}
                                    to="/about"
                        >
                                    Yee Wai
                        </Link>
            </div>
            <div
                        className="col"
                        id="code"
            >
                        <Link
                                    className="nav animated fadeInUp"
                                    replace={false}
                                    to="/code"
                        >
                                    Code
                        </Link>
            </div>
</div>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col"
                id="art"
>
                <Link
                                className="nav animated fadeInUp"
                                replace={false}
                                to="/art"
                >
                                Art
                </Link>
</div>,
              <div
                className="col-2"
                id="about"
>
                <Link
                                className="nav animated fadeIn animation-delay-1000"
                                replace={false}
                                to="/about"
                >
                                Yee Wai
                </Link>
</div>,
              <div
                className="col"
                id="code"
>
                <Link
                                className="nav animated fadeInUp"
                                replace={false}
                                to="/code"
                >
                                Code
                </Link>
</div>,
            ],
            "className": "row justify-content-around",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="nav animated fadeInUp"
                  replace={false}
                  to="/art"
>
                  Art
</Link>,
                "className": "col",
                "id": "art",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Art",
                  "className": "nav animated fadeInUp",
                  "replace": false,
                  "to": "/art",
                },
                "ref": null,
                "rendered": "Art",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="nav animated fadeIn animation-delay-1000"
                  replace={false}
                  to="/about"
>
                  Yee Wai
</Link>,
                "className": "col-2",
                "id": "about",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Yee Wai",
                  "className": "nav animated fadeIn animation-delay-1000",
                  "replace": false,
                  "to": "/about",
                },
                "ref": null,
                "rendered": "Yee Wai",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="nav animated fadeInUp"
                  replace={false}
                  to="/code"
>
                  Code
</Link>,
                "className": "col",
                "id": "code",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Code",
                  "className": "nav animated fadeInUp",
                  "replace": false,
                  "to": "/code",
                },
                "ref": null,
                "rendered": "Code",
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "nav",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
